src/main.rs

```diff
@@ -12,13 +14,16 @@ use crate::cli::CliArguments;
 
 mod cli;
 
-fn get_input(input: Option<&PathBuf>) -> String {
+fn get_input(input: Option<&PathBuf>) -> Result<String> {
     match input {
-        Some(path) => std::fs::read_to_string(path).unwrap(),
+        Some(path) => std::fs::read_to_string(path)
+            .with_context(|| format!("failed to read the file {:#?} into string", path)),
```

Comment:
```suggestion
            .with_context(|| format!("failed to read file {:#?}", path)),
```

BTW, what does the output looks like? `{:#?}` looks strange here